test/e2e/serial/tests/e2e_suite.go

```diff
@@ -47,6 +48,11 @@ import (
 	e2ewait "github.com/openshift-kni/numaresources-operator/test/utils/objects/wait"
 )
 
+const (
```

Comment:
Why not read these values from the nodes themselves under the node ```.status.allocatable``` and ```.status.capacity```

Reply:
because it will not there. We need to perform the same logic that is done when we run `oc described node` to get the stats at the end; IOW, we need to sum the `requests` of all the pods running on a node, is not a simple difference.
It is surely something we can improve later though.

Reply:
fixed in the last commit